src/utils/languages.js

```diff
@@ -1440,6 +1440,21 @@ const languages = [
     canonical_name: 'Herero',
     code: 'hz',
     name: 'Otsiherero'
+  },
+  {
+    canonical_name: 'Maithili',
+    code: 'mai',
+    name: '\u092E\u0948\u0925\u093F\u0932\u0940'
```

Comment:
Is there a specific need to use unicode escapes? Most JSON parsers can deal with Unicode.

Reply:
not really, the initial list is given in the Unicode escape form, therefore we are doing the same here.